src/core/ProjectRenderer.cpp

```diff
@@ -124,6 +124,15 @@ ProjectRenderer::ExportFileFormats ProjectRenderer::getFileFormatFromExtension(
 
 
 
+QString ProjectRenderer::getFileExtensionFromFormat(
+		ExportFileFormats _fmt )
```

Comment:
In all future code added to lmms, arguments should not be prefixed with `_` - as I understand, that is an old convention that is being phased out.